charts/workspacemanager/templates/_helpers.tpl

```diff
@@ -37,3 +37,59 @@ Service firewall
   {{- end -}}
   {{- end -}}
 {{- end }}
+
+{{/*
+LDAP base domain template
+*/}}
+{{- define "workspacemanager.ldapbasedomain" -}}
+  dc=dsde-{{ .Values.global.terraEnv }},dc=broadinstitute,dc=org
```

Comment:
We are trying to steer away from using global.terraEnv in string interpolations and instead using explicit values. Using `global.terraEnv` assumes/forces certain naming conventions and makes these charts less portable. For example, in this case I would recommend replacing this template with a `.Values.proxy.ldap.baseDomain` value. This will cause our values overrides to become more verbose, but we think it's worth it in the end.